packages/beacon-node/src/api/impl/beacon/state/index.ts

```diff
@@ -202,7 +202,14 @@ export function getBeaconStateApi({
 
       const epoch = filters.epoch ?? computeEpochAtSlot(state.slot);
       const startSlot = computeStartSlotAtEpoch(epoch);
-      const shuffling = stateCached.epochCtx.getShufflingAtEpoch(epoch);
+      const decisionRoot = stateCached.epochCtx.getDecisionRoot(epoch);
```

Comment:
```suggestion
      const decisionRoot = stateCached.epochCtx.getShufflingDecisionRoot(epoch);
```
this is more clear to me

Reply:
Updated.